### PR TITLE
New package: fs-uae-launcher-3.0.5.

### PR DESCRIPTION
**Pull request by Veganizer**
#### General
- [x] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [ ] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [x] I generally don't use the affected packages but briefly tested this PR

#### Does it build and run successfully? 
(Please choose at least one native build and, if supported, at least one cross build. More are better.)
- [x] I built this PR locally for my native architecture, (x86-64-musl)
- [ ] I built this PR locally for these architectures (if supported. mark crossbuilds):
  - [ ] aarch64-musl
  - [ ] armv7l
  - [ ] armv6l-musl

### CODE REVIEW

**Comment by Johnnynator** [comment]
```
/usr/share/fs-uae-launcher/OpenGL/DLLS/glut32.dll
/usr/share/fs-uae-launcher/OpenGL/DLLS/glut64.dll
/usr/share/fs-uae-launcher/OpenGL/DLLS/glut_README.txt
/usr/share/fs-uae-launcher/OpenGL/DLLS/opengle32.vc7.dll
/usr/share/fs-uae-launcher/OpenGL/DLLS/opengle32.vc9.dll
/usr/share/fs-uae-launcher/OpenGL/DLLS/opengle64.vc9.dll
```
are these dll's actually needed? Probably not?

File: srcpkgs/fs-uae-launcher/template [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'fs-uae-launcher'
+pkgname=fs-uae-launcher
+version=3.0.5
+revision=1
+build_style=gnu-makefile
+make_install_args="prefix=/usr"
+hostmakedepends="gettext python3"
+depends="fs-uae python3 python3-Pillow python3-PyQt5 python3-PyQt5-opengl python3-requests"
```
Comment (ericonr) [inline srcpkgs/fs-uae-launcher/template:8]: Doesn't need to depend on python if it depends on python libraries.

**Review by ericonr — COMMENTED** [review]
short_desc="GUI for the FS-UAE Amiga emulator"

I'd consider removing the "Amiga emulator" part even.

**Comment by Justinizer** [comment]
> ```
> /usr/share/fs-uae-launcher/OpenGL/DLLS/glut32.dll
> /usr/share/fs-uae-launcher/OpenGL/DLLS/glut64.dll
> /usr/share/fs-uae-launcher/OpenGL/DLLS/glut_README.txt
> /usr/share/fs-uae-launcher/OpenGL/DLLS/opengle32.vc7.dll
> /usr/share/fs-uae-launcher/OpenGL/DLLS/opengle32.vc9.dll
> /usr/share/fs-uae-launcher/OpenGL/DLLS/opengle64.vc9.dll
> ```
> 
> are these dll's actually needed? Probably not?

```bash
cat /usr/share/fs-uae-launcher/OpenGL/DLLS/GLE_WIN32_README.txt 
This is a Win32 version of the GLE library
compiled as a shared library (DLL) for use with
e.g. Python's ctypes foreign-function interface.

To install, copy the file gle32.dll to your system32
```
Seems not to be needed I will try to cut it out somehow...

**Pull request closed without merging**